src/components/DataMessenger/DataMessenger.js

```diff
@@ -1299,17 +1299,16 @@ export default class DataMessenger extends React.Component {
       'drawer-content-wrapper'
     )[0]
     if (
-      (chartToolTipElement && dataMessenger) ||
-      this.props.placement !== 'top' ||
+      chartToolTipElement &&
+      dataMessenger &&
+      this.props.placement !== 'top' &&
```

Comment:
this placement prop can only be either top or bottom, never both. changing || to && will nullify the whole condition